modules/serverless-ossm-jwt-existing.adoc

```diff
@@ -0,0 +1,115 @@
+// Module included in the following assemblies:
+// * serverless/networking/serverless-ossm-jwt.adoc
+
+[id="serverless-ossm-jwt-existing_{context}"]
+= Enable sidecar injection for an existing Knative service
```

Comment:
`Create a Knative service with sidecar injection` and `Enable sidecar injection for an existing Knative service` are almost same.
 
https://jwt--ocpdocs.netlify.app/openshift-enterprise/latest/serverless/networking/serverless-ossm-jwt.html#serverless-ossm-jwt-existing_serverless-ossm-jwt

I wonder that `Enable sidecar injection for an existing Knative service` is a little bit redundant.

Reply:
On a second thought, I agree with @nak3 that the whole section `Enable sidecar injection for an existing Knative service` is a bit redundant. We could probably replace it with one or two sentences in the `Create a Knative service with sidecar injection` section. Something like this:
`Alternatively, enable sidecar injection for an existing service by adding the annotation sidecar.istio.io/inject: "true". Edit the existing service using oc edit ksvc <name>`

Reply:
I can understand why it seems redundant, but I'm not sure what's the best way to approach this in terms of user story, i.e. if the procedure for enabling this on a new vs an existing service are different, it should be called out so that a user can find that easily. I'm worried that if we add this as just a note or something, someone would follow the steps for a new service, then realize after that enabling it for an existing service is different?

Reply:
OK. Understood. ...We repeat a lot of stuff. How about this:
* The first heading would be `Enable sidecar injection`
* Then we'd have two sub-headers: `Create a new Knative service with sidecar injection`, `Enable sidecar injection for an existing Knative service`.
The first sub-section would include just steps 1., 2. from https://jwt--ocpdocs.netlify.app/openshift-enterprise/latest/serverless/networking/serverless-ossm-jwt.html#serverless-ossm-jwt-new-service_serverless-ossm-jwt 
The second sub-section could either include points 1., 2. from https://jwt--ocpdocs.netlify.app/openshift-enterprise/latest/serverless/networking/serverless-ossm-jwt.html#serverless-ossm-jwt-existing_serverless-ossm-jwt or it could mention the `oc edit ksvc <name>` and adding the same annotations as in the previous step.
All of this would be followed by step 3. and verification steps. 
We would't need to repeat step 3. and verification steps twice and it would make it more concise. 
If this doesn't work for you then let's keep what we have now

Reply:
I'll keep what we have for now just because that's how it should be in terms of the mod doc guidelines I think, i.e. one full procedure that inc. verification steps for the procedure.
We can get some opinions / feedback from the OCP docs folks and see if they think there's a better way that keeps our doc guidelines during that review.